packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5138,7 +5140,74 @@ export default class Meeting extends StatelessWebexPlugin {
     }
   };
 
+  /**
+   * Handles an incoming Roap message
+   * @internal
+   * @param {RoapMessage} roapMessage roap message
+   * @returns {undefined}
+   */
+  public roapMessageReceived = (roapMessage: RoapMessage) => {
```

Comment:
this function is just code moved from meetings util

Reply:
Is there any reason why we can't now move this into the roap folder and remove roap references (except for the one roap object) from meeting/index?

Reply:
I'm not convinced if that belongs better in the roap folder. The code in SDK's roap folder now is only dealing with preparing the http requests for Locus when sending roap messages and handling the responses, it doesn't even reference mediaProperties or webrtcMediaConnection anywhere. Meeting object is the one that knows about the media connection it has and the roap messages need to be forwarded to it.
There is also other code related to roap in meeting/index - see inside setupMediaConnectionListeners(). Maybe we could look at moving all of this into roap folder... not sure...